medicines/web/cypress/integration/tests.js

```diff
@@ -0,0 +1,84 @@
+  win.eval(polyfill);
+  win.fetch = win.unfetch;
+
+  // Clear out session storage so that the disclaimer is always presented.
+  win.sessionStorage.clear();
+})
+
+describe('Search', function() {
+  it('Search for Paracetamol', function() {
+    cy.server();
+    // Mock out first page of search results.
+    cy.route(
+      'https://mhraproductsdev.search.windows.net/indexes/products-index/docs?api-key=CFBCBE8AA11AA871C14001527533870C&api-version=2017-11-11&highlight=content&queryType=full&$count=true&$top=10&$skip=0&search=paracetamol~1+paracetamol^4&scoringProfile=preferKeywords',
+      'fixture:search_results.json',
+    );
+    // Mock out second page of search results.
+    cy.route(
+      'https://mhraproductsdev.search.windows.net/indexes/products-index/docs?api-key=CFBCBE8AA11AA871C14001527533870C&api-version=2017-11-11&highlight=content&queryType=full&$count=true&$top=10&$skip=10&search=paracetamol~1+paracetamol^4&scoringProfile=preferKeywords',
+      'fixture:search_results.json',
+    );
+
+    cy.visit('/');
+    cy.get("input[type='search']").type('paracetamol');
+    cy.contains('Search').click();
+    cy.contains('I have read and understand the disclaimer').click();
+    cy.contains('Agree').click();
+    cy.contains('Next').click();
+    cy.get("a[href='https://example.com/my-cool-document.pdf']");
+  });
+});
+
+describe('A-Z Index', function() {
+  it('Navigate to Paracetamol via A-Z index', function() {
+    cy.server();
+    // Mock out list of substances and medcines.
+    cy.route(
+      "https://mhraproductsdev.search.windows.net/indexes/products-index/docs?api-key=CFBCBE8AA11AA871C14001527533870C&api-version=2017-11-11&facet=facets,count:50000,sort:value&$filter=facets/any(f:+f+eq+'P')&$top=0",
+      'fixture:facets.json',
+    );
+    // Mock out first page of search results.
+    cy.route(
+      'https://mhraproductsdev.search.windows.net/indexes/products-index/docs?api-key=CFBCBE8AA11AA871C14001527533870C&api-version=2017-11-11&highlight=content&queryType=full&$count=true&$top=10&$skip=0&search=PARACETAMOL~1+PARACETAMOL^4+TABLETS~1+TABLETS^4&scoringProfile=preferKeywords',
+      'fixture:search_results.json',
+    );
+    // Mock out second page of search results.
+    cy.route(
+      'https://mhraproductsdev.search.windows.net/indexes/products-index/docs?api-key=CFBCBE8AA11AA871C14001527533870C&api-version=2017-11-11&highlight=content&queryType=full&$count=true&$top=10&$skip=10&search=PARACETAMOL~1+PARACETAMOL^4+TABLETS~1+TABLETS^4&scoringProfile=preferKeywords',
+      'fixture:search_results.json',
+    );
+
+    cy.visit('/');
+    cy.get('nav')
+      .contains('P')
+      .click();
+    cy.contains('PARACETAMOL').click();
+    cy.contains('PARACETAMOL TABLETS').click();
+    cy.contains('I have read and understand the disclaimer').click();
+    cy.contains('Agree').click();
+    cy.contains('Next').click();
+    cy.get("a[href='https://example.com/my-cool-document.pdf']");
```

Comment:
This doesn't represent a user action. A user would click on text, and find that it lets them download/view a PDF. They wouldn't look for an anchor tag based on already knowing the `href`.